censustree/censustree.go

```diff
@@ -18,10 +16,8 @@ import (
 )
 
 var (
-	censusWeightKey         = []byte("censusWeight")
-	censusIndexKey          = []byte("censusIndex")
-	isIndexAsKeysCensus     = []byte("isIndexAsKey")
-	censusKeysToIndexPrefix = []byte("keyIndex")
+	censusWeightKey = []byte("censusWeight")
+	censusIndexKey  = []byte("censusIndex")
```

Comment:
Is this DB key necessary now?

Reply:
the key and also functions [censustree.updateCensusIndex()](https://github.com/vocdoni/vocdoni-node/blob/feature/indexed_census_deprecation/censustree/censustree.go#L223) and [censustree.GetCensusIndex()](https://github.com/vocdoni/vocdoni-node/blob/feature/indexed_census_deprecation/censustree/censustree.go#L357) no?